Add principles mission (#217)
publiccodenet/about: organization/mission.md

type: Resource
---

# Mission & founding principles

## Mission

> To enable public-purpose software and policy that is open and collaborative.

Reviewer: Arnout's suggestion:
```suggestion
To make public-purpose software open and collaborative.
```

Author: I don't think this will be useful.

First off we don't _make_ anything, and don't have the ambition too since it would lead to disempowerment.

Secondly I think undervaluing the role of policy in the digital transformation and effectively seperating it from software is the whole reason we're in this mess, so I think for us to be able to execute on our mission it is crucial that we include policy at the highest level, and software as well.

Was there any more reasoning?
